### PR TITLE
Add organizer dashboard

### PR DESCRIPTION
**Pull request by ruilopesm**
Needed by [#129](https://github.com/coderdojobraga/shuriken/pull/129) from shuriken.

### CODE REVIEW

File: lib/bokken/accounts.ex [hunk 1]
```diff
@@ -215,6 +259,36 @@ defmodule Bokken.Accounts do
     Repo.delete_all(query)
   end
 
+  def update_mentor_and_user(mentor_params, user_params) do
```
Comment (MarioRodrigues10) [inline lib/bokken/accounts.ex]: Make this handle more error cases such has ```Mentor and user not found``` and ```An error occurred while fetching the user or mentor```

File: lib/bokken/accounts.ex [hunk 2]
```diff
@@ -380,6 +456,36 @@ defmodule Bokken.Accounts do
     end
   end
 
+  def update_ninja_and_user(ninja_params, user_params) do
```
Comment (MarioRodrigues10) [inline lib/bokken/accounts.ex]: Make this handle more error cases such has ```Ninja and user not found``` and ```An error occurred while fetching the user or mentor```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Review by MarioRodrigues10 — DISMISSED** [review]
Good Job ! :muscle:

File: lib/bokken_web/controllers/admin_controller.ex [hunk 3]
```diff
@@ -0,0 +1,42 @@
+defmodule BokkenWeb.AdminController do
```
Comment (MarioRodrigues10) [inline lib/bokken_web/controllers/admin_controller.ex]: Protect the endpoints please with ```when is_organizer(conn)```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Also there's a bug when i try to fetch mentors, ninjas, guardians in frontend
```c
 function AdminView.__resource__/0 is undefined
```

**Review by MarioRodrigues10 — DISMISSED** [review]
LGTM ! :rocket:

**Comment by MarioRodrigues10** [comment]
There's a 500 error when we try to use the endpoint to fetch mentors on the route ``` /admin/lectures/new ```.
Please try to fix that :pray:

**Review by MarioRodrigues10 — DISMISSED** [review]
Good Job !

**Review by MarioRodrigues10 — DISMISSED** [review]

**Review by MarioRodrigues10 — DISMISSED** [review]

File: lib/bokken_web/router.ex [hunk 4]
```diff
@@ -32,6 +31,19 @@ defmodule BokkenWeb.Router do
   scope "/api", BokkenWeb do
     pipe_through :api
 
+    scope "/admin" do
+      pipe_through [:authenticated, :admin]
+
+      get "/users", AdminController, :index_users
```
Comment (nelsonmestevao) [inline lib/bokken_web/router.ex]: I think you should respect REST. And split the controller into smaller modules.

```elixir
scope "/users", AdminController do
  get "/", User, :index
  put "/", User, :update
end
```

If you think the endpoints are really simple, you can even define it inside the same controller file. Like this:

```elixir
defmodule AdminController do
  defmodule User do
  ...
  end
  
  defmodule Ninja do
  ...
  end
  ...
```

Comment (nelsonmestevao) [reply lib/bokken_web/router.ex]: Basically, I think you should be able to use the [`resources`](https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4) macro.

```elixir
resources "/user", AdminController.User, only: [:index, :update]
```

Comment (ruilopesm) [reply lib/bokken_web/router.ex]: It's ironic that the branch name contains "rest" and I'm not implementing it 😅
Will implement it 🙏🏼

**Review by ruioliveira02 — COMMENTED** [review]
Add the new endpoints to Postman pls :pray:

**Review by ruioliveira02 — COMMENTED** [review]
Just add the endpoints to Postman :pray: We are refactoring the tests anyway, so I'm not too bothered by them not being here

Also, could you bump the version in `mix.exs` to `1.0.6` pls?

**Review by ruioliveira02 — APPROVED** [review]

**Pull request merged**